Hazel/MessageReader.cs

```diff
@@ -256,21 +249,6 @@ public uint ReadPackedUInt32()
             return output;
         }
 
-        public T ReadNetObject<T>(IGame game) where T : IInnerNetObject
-        {
-            return game.FindObjectByNetId<T>(ReadPackedUInt32());
-        }
-
-        public Vector2 ReadVector2()
```

Comment:
moved to an impostor-specific extension method?

Reply:
[yup](https://github.com/Impostor/Impostor/blob/net7-nugethazel/src/Impostor.Api/Net/MessageReaderExtensions.cs)